src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -354,24 +354,25 @@ public ReadableStreamPullState(Stream stream, CancellationToken cancellationToke
             _buffer = new byte[65536];
         }
 
-        public async Task Pull()
+        public async Task PullAsync()
         {
             try
             {
-                Memory<byte> view = _buffer;
-                int length = await _stream.ReadAsync(view, _cancellationToken).ConfigureAwait(true);
-                using (Buffers.MemoryHandle handle = view.Pin())
-                {
-                    ReadableStreamControllerEnqueueUnsafe(this, handle, length);
-                }
+                int length = await _stream.ReadAsync(_buffer, _cancellationToken).ConfigureAwait(true);
             }
             catch (Exception ex)
             {
                 BrowserHttpInterop.ReadableStreamControllerError(this, ex);
             }
         }
-        private static unsafe void ReadableStreamControllerEnqueueUnsafe(object pullState, Buffers.MemoryHandle handle, int length) =>
-            BrowserHttpInterop.ReadableStreamControllerEnqueue(pullState, (IntPtr)handle.Pointer, length);
+
+        private unsafe void ReadableStreamControllerEnqueueUnsafe(object pullState, int length)
```

Comment:
Looks like this method isn't getting called?

Reply:
![image](https://github.com/dotnet/runtime/assets/271576/1b1c06c1-c670-421e-b3b3-c595ece48624)

Reply:
because `type: "bytes"` need to be there.
Now enqueue is not failing, but `http_wasm_readable_stream_controller_enqueue` is never called with zero

Reply:
For me it is working with this fix https://github.com/dotnet/runtime/pull/91295/commits/dd83582568b482dd5dda7da0899ce1181fb9e1bd . Depending on which tests you run it might take a while to get to `0` as the send a lot of data.